components/cd-card/cd-card.html

```diff
@@ -0,0 +1,156 @@
+<link rel=stylesheet href="../cd-base.css" type="text/css" media=screen>
+<link rel=stylesheet href="../cd-grid/cd-grid.css" type="text/css" media=screen>
+<link rel=stylesheet href="../cd-flow/cd-flow.css" type="text/css" media=screen>
+<link rel=stylesheet href="../cd-utilities/cd-utilities.css" type="text/css" media=screen>
+<link rel=stylesheet href="../cd-typography/cd-typography.css" type="text/css" media=screen>
+<link rel=stylesheet href="../cd-button/cd-button.css" type="text/css" media=screen>
+<link rel=stylesheet href="../cd-read-more/cd-read-more.css" type="text/css" media=screen>
+<link rel=stylesheet href="cd-card.css" type="text/css" media=screen>
+
+<div class="cd-layout-container">
+  <div class="cd-container cd-grid cd-grid-3-col">
+
+    <article role="article" class="cd-card node--view-mode-card [ cd-flow ]">
+
+      <div class="cd-card__container [ cd-flow ]">
+
+        <div class="cd-card__title">
+          <a href="#" rel="bookmark">Sed dapibus lectus sit amet adipiscing egestas mauris est viverra nibh</a>
+        </div>
+
+        <div class="cd-card__description node__content">
```

Comment:
`node__content` also doesn't get referenced in the CSS. I feel like removing these is the safe move, to avoid having something from Drupal come along and pop some styles onto the card accidentally.

Reply:
These classes will be in other content-related components too. I'll remove the classes throughout anda also the `.node--view-mode` from the comment above.
I originally included them to make the mental link between the specific component and what we are used to in Drupal, but that isn't necessary.